### PR TITLE
Angular 13 & rxjs 7

### PR DESCRIPTION
**Pull request by hheexx**
Interested?
Watch out for changed package name and versions.

### CODE REVIEW

**Comment by HarelM** [comment]
Thanks!!!
I want to finilize maplibre 12 with all the typescript changes first and then create a version with a "good" maplibre version, and then I'll merge this.
Can you change the version of this package to 13.0.0 so that when I merge all I will have to do is click the publish button?

**Comment by hheexx** [comment]
Contact me when you plan to merge, I will fix it.
One more thing to consider is rxjs. I updated here to rxjs 7 but i'm not sure what that implies....

**Comment by HarelM** [comment]
Did you use some tool to upgrade? I might want to update my project to angular 13 to test this as well...

**Comment by hheexx** [comment]
npx ng update everything. It has schematics that made some thanges.
Then I updated remaining libs manualy in package.json

**Comment by hheexx** [comment]
angular material and angular forms removed as it's not used....

**Comment by HarelM** [comment]
Seems like the tests are failing.
I think the showcase is using material so it shouldn't be removed...?

**Comment by HarelM** [comment]
Would you be integrated in helping me maintain this library?
I moved to angular 13 too in my project, so this is going to happen soon I guess :-)

**Comment by hheexx** [comment]
Ok, I will bring it back.
I could help with a lib....

File: projects/ngx-maplibre-gl/package.json
Comment (HarelM) [inline projects/ngx-maplibre-gl/package.json:2]: Nope...

File: projects/ngx-maplibre-gl/src/lib/source/geojson/feature.component.ts
Comment (HarelM) [inline projects/ngx-maplibre-gl/src/lib/source/geojson/feature.component.ts:51]: Better to send undefined?

**Review by HarelM — CHANGES_REQUESTED** [review]
Also I guess you are using node 16, I think it will be better to do it in another commit...

**Comment by hheexx** [comment]
> Also I guess you are using node 16, I think it will be better to do it in another commit...

How you see that?

**Comment by HarelM** [comment]
The changes in the lock file

**Comment by HarelM** [comment]
I've activated the tests again as they were not ruining for some reason. Make sure to fix the failing tests as well.

**Comment by hheexx** [comment]
> The changes in the lock file

Aha. It;s not node related but npm related. npm versions >6 create it but all recent  versions of npm@6 also read it.
try npm i npm -g

**Comment by HarelM** [comment]
The errors message is a bit strange. Can you check if these tests pass locally?

**Comment by hheexx** [comment]
> The errors message is a bit strange. Can you check if these tests pass locally?
yea strange. I dont have chrome, i tried e2e tests with firefox and everything look ok.

**Comment by HarelM** [comment]
Can you try and rebase? I think I managed to fix the failing tests.

**Comment by HarelM** [comment]
I've created a branch for node 16. I'll merge it soon. I still don't know what's wrong with the language switch test as it passes locally on my machine. but I'm using mac and not ubuntu. I'll need to dig into it, but that shouldn't hold this PR.
I'll work on it in the next few days.
Thanks for tolerating my annoying comments :-)
Let me know if you would like to help maintain this repo as currently I'm the only maintainer which is not ideal...

**Comment by hheexx** [comment]
Thanks!
Yea I'm OK with dedicating some time for this....
ill keep it updated at least.

**Comment by HarelM** [comment]
I've added you with write permissions.

**Comment by hheexx** [comment]
so I merge?

**Comment by HarelM** [comment]
Can you rebase? I've updated to node 16 in another branch.

**Review by HarelM — APPROVED** [review]

**Review by HarelM — APPROVED** [review]

**Comment by HarelM** [comment]
Thanks!! Any chance you can see if you can solve the the failing tests?
Not related to this PR of course...

**Pull request merged**
